das/aggregator.go

```diff
@@ -41,6 +41,8 @@ var DefaultAggregatorConfig = AggregatorConfig{
 	DumpKeyset:    false,
 }
 
+var BatchToDasFailed = errors.New("unable to batch to DAS, falling back to storing data on chain")
```

Comment:
I think we want to change this message since a batch failing doesn't always fall back to chain if it's disabled

Reply:
Done.